test/StoreitValueTest.js

```diff
@@ -157,6 +157,14 @@ describe("StoreitValue", function () {
                     this.value.on("changed", this.changedListener);
                 });
 
+                it("should listen to the store for modified events", () => {
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>Too many errors. (50% scanned).